Source/AwsCommonRuntimeKit/io/Stream.swift

```diff
@@ -35,7 +47,7 @@ extension FileHandle: AwsInputStreamImpl {
         let savedPos = self.offsetInFile
         defer { self.seek(toFileOffset: savedPos ) }
         self.seekToEndOfFile()
-        // TODO: What happens if the UInt64 is > Int64.max ??
+        //What happens if the UInt64 is > Int64.max ??
```

Comment:
You will get failures everywhere, all file I/O. Files simply can't be bigger than that. Also, no hard drive can hold a file that size.

Reply:
henson wrote this comment, left it there cuz didn't know if I needed to worry about it or not lol

Reply:
removing this old comment